src/components/biblio/topic_entity_tag/TopicEntityCreate.js

```diff
@@ -66,16 +67,20 @@ const TopicEntityCreate = () => {
     
   const [curieToNameTaxon, setCurieToNameTaxon] = useState({});
   const [modToTaxon, setModToTaxon] = useState({});
+  const [taxonToMod, setTaxonToMod] = useState({});
```

Comment:
@ianlongden : I think @sweng66 's function to remap modToTaxon -> taxonToMod would go here.  Modified to loop through every mod's taxon instead of just the first taxon